scripts/install-windows-prereqs.ps1

```diff
@@ -662,6 +662,10 @@ try {
     Install-Node
     Install-VCRuntime
 
+    Set-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem" `
```

Comment:
Could you add a comment detailing that this is for the long path issue encountered in ci? It's not obvious, unfortunately.

Reply:
Yes. I'll do this.